scipy/interpolate/_rgi.py

```diff
@@ -371,17 +374,22 @@ def _evaluate_spline(self, values, xi, spline_degree):
             xi = xi.reshape((1, xi.size))
         m, n = xi.shape
 
+        if spline_degree == 'pchip':
```

Comment:
Ah sorry. My review was not good. As @tylerjereddy found, this line is always false because spline_degree is a number (maybe 3 for both pchip and cubic. Maybe we need to pass `method` to the `_evaluate_spline` and use it here.